modules/learn/pages/services-and-indexes/services/backup-service.adoc

```diff
@@ -83,6 +83,14 @@ Optionally, the data itself can be retained, on the local filesystem.
 A deleted repository whose data still exists can be _imported_ back into the cluster, if required.
 Once imported, the repository can be _read_ from, but no longer receives data backups.
 
+[#avoiding-task-overlap]
+== Avoiding Task Overlap
+
+Although the Backup Service allows automated tasks to be scheduled for repetition at intervals as small as one minute, administrators are recommended typically not to lower the interval below fifteen minutes; and always to ensure that the established interval allows each scheduled task-repetition ample time to complete before the next; even in the event of unanticipated, high network-latency.
+
+If, due to an interval-specification that is too small, a given task attempts to start while its prior repetition is still ongoing, the new task fails, stops running, and is not restarted.
+In such circumstances, all data continues to be backed up by the prior repetition; and will be backed up again by any subsequent repetition that starts after the completion of the prior repetition.
```

Comment:
In such circumstances, all data continues to be backed up by the prior repetition; and will be backed up again by any subsequent repetition that starts after the completion of the previous backup.

Reply:
ok